src/components/avatar.vue

```diff
@@ -86,16 +86,16 @@ export default defineComponent({
 });
 </script>
 <style scoped>
-.profile-wrap {
+.profile-wrap { width: 100% !important; height: auto !important;
   position: relative;
   display: inline-block;
 }
 .letter-avatar {
   border-radius: 100%;
   display: inline-block;
   border: 2px solid #aaa;
-  width: 100%;
-  height: 100%;
+  width: 75%;
+  height: 75%;
 }
```

Comment:
This changes the size of the avatar _everywhere_ have you checked this has no negative side-effects on other components?